README.md

```diff
@@ -20,3 +20,20 @@ Project](https://chipsalliance.org/), a Series of LF Projects, LLC.
 
 The [Caliptra WorkGroup Technical Charter](Caliptra_WG_Technical_Charter.md)
 sets out the charter governing the Caliptra project.
+
+## Repositories
+
+The Caliptra code base and documentation are split across five repositories:
+
+| Repo | Where |
+| -- | -- |
+| [Caliptra RTL](https://github.com/chipsalliance/caliptra-rtl) | The primary repository with immutable RTL |
+| [Caliptra SW](https://github.com/chipsalliance/caliptra-sw) | Common libraries and tools shared by Firmware, FMC and ROM |
```

Comment:
Caliptra firmware (ROM, FMC, Runtime), and libraries/tools needed to build and test firmware.

Reply:
applied

Reply:
Are you sure? I don't see the change.

Reply:
https://github.com/chipsalliance/Caliptra/compare/main...repos